guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,237 @@
+"""
+
+"""shell
+!pip install -q --upgrade git+https://github.com/keras-team/keras-cv.git tensorflow
+"""
+
+import json
+import keras_cv
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import keras
+import numpy as np
+
+"""
+## Inference with a pretrained classifier
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_beginner.png)
+
+Let's get started with the simples KerasCV API: a pretrained classifier.
+In this example we will build a Dogs vs Cats classifier using a model that was
+pretrained on the ImageNet dataset.
+
+The highest level module in KerasCV is a *task*. A *task* is a `keras.Model`
+consisting of a (generally pretrained) backbone model and task-specific layers.
+Here's an example using keras_cv.models.ImageClassifier with a ResNet50V2
+Backbone.
+
+
+"""
+
+import json
+import keras_cv
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import keras
+import numpy as np
+
+# TODO(lukewood): replace with ImageClassifier.from_preset()
+classifier = keras_cv.models.DenseNet121(
+    weights='imagenet',
+    include_top=True,
+    include_rescaling=True,
+    num_classes=1000
+)
+
+filepath = tf.keras.utils.get_file(origin="https://i.imgur.com/9i63gLN.jpg")
+image = keras.utils.load_img(filepath)
+
+"""
+![Picture of a cat with a velvet blanket](https://i.imgur.com/9i63gLN.jpg)
+"""
+
+image = np.array(image)
+
+predictions = classifier.predict(image[None, ...])
+top_classes = predictions[0].argsort(axis=-1)
+
+# Subset of imagenet classes
+classes = {
+    281: 'tabby, tabby cat',
+    885: 'velvet'
+}
+top_two = [classes[i] for i in top_classes[-2:]]
+print("Top two classes are:", top_two)
+
+"""Great!  Both of these are correct!  But what if you don't care about the
+velvet blanket, and instead only want to know if a cat is in the image or not?
+This can be solved using fine tuning.
+
+# Fine tuning a pretrained classifier
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_intermediate.png)
+
+When labeled images specific to our task are available, fine-tuning a custom
+classifier can improve performance. If we want to train a Cats VS Dogs
+Classifier, using explicitly labeled Cat vs Dog data should perform better than
+the generic classifier data! And for many tasks, no relevant pretrained model
+will be available (e.g., categorizing images specific to your application).
+
+The biggest difficulty when finetuning a KerasCV model is loading and augmenting
+your data.  Luckily, we've handled the second half for you, so all you'll have
+to do is load your own data.
+
+First, let's setup our data pipeline:
+"""
+
+BATCH_SIZE = 32
+AUTOTUNE = tf.data.AUTOTUNE
+tfds.disable_progress_bar()
+
+data, dataset_info = tfds.load("cats_vs_dogs", with_info=True, as_supervised=True)
+train_steps_per_epoch = dataset_info.splits["train"].num_examples // BATCH_SIZE
+train_dataset = data['train']
+
+IMAGE_SIZE = (224, 224)
```

Comment:
Is there any harm in inling everything before `package_dict`? Nothing is reused and it would make the code easier to read.

Reply:
package_dict() could be reused for evaluation; perhaps I should add that back in